core/components/molecules/modal/__stories__/style.css

```diff
@@ -0,0 +1,3 @@
+.Modal-list {
+    height: 185px;
```

Comment:
Please use the token instead. The nearest token to this value is `var(--spacing-8)`.